plugins/uefi-capsule/fu-uefi-device.h

```diff
@@ -84,6 +84,12 @@ typedef enum {
  * Do not prepend a plausible missing capsule header.
  */
 #define FU_UEFI_DEVICE_FLAG_NO_CAPSULE_HEADER_FIXUP (1 << 7)
+/**
+ * FU_UEFI_DEVICE_FLAG_ENABLE_DEBUGGING:
+ *
+ * Increase debugging level for the device.
```

Comment:
`Enable debugging the EFI binary` perhaps

Reply:
WFM, thanks.